### PR TITLE
159 nicer error views

### PR DESCRIPTION
**Pull request by jLebioda**

### CODE REVIEW

File: web/templates/error.html [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+                    </p>     
+                    <p>
+                        In practice it usually means, that filling the form took you slightly too long time (more than an hour). <strong>Go back and try to fill the form once again, please.</strong>
+                    </p>
+                    <p>
+                        If you see this page in effect of clicking on any link you received from someone via mail or any other message, please <a href="mailto:lcsb.sysadmins@uni.lu?subject=DAISY+error">let us know</a>.
+                    </p>
+                {% elif reason == "500" %}
+                    <p class="display-4">
+                        <strong>The application has encountered the error.</strong>
```
Comment (vildead) [inline web/templates/error.html:55]: "an error" instead of "the error"?

File: web/templates/error.html [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+                {% if reason == "404" %}
+                    <p class="display-4">
+                        <strong>The page does not exist. Let us guide you back!</strong>
+                    </p>
+                    <p>
+                        Maybe you received a link to an obsolete resource or didn't copy the whole link?
+                    </p>
+                    <p>
+                        You might want to:
+                        <ul>
```
Comment (vildead) [inline web/templates/error.html:25]: this block can be shown for all errors. Would it make sense?

File: web/templates/error.html [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+                    </p>
+                    <p>
+                        It might have happened because of the data you entered. If you'd like, you can let us know.
+                    </p>
+                {% elif reason == "403" %}
+                    <p class="display-4">
+                        <strong>The page you requested is restricted, and your user cannot access the resource.</strong> 
+                    </p>
+                    <p>
+                        If you are sure that you should be able to access the document, please contact your Data Steward.
```
[the remark this answers is not in the document]
Comment (vildead) [reply web/templates/error.html:55]: you are right - lets keep it as it is :)
 the role of data steward is not (yet) defined in Daisy anyway.

**Review by vildead — APPROVED** [review]

**Comment by jLebioda** [comment]
I will test the changes on one of the VMs before merging

**Comment by jLebioda** [comment]
@vildead 500 handler should be OK now. Would you mind testing on a VM, please?

**Pull request merged**